src/main/java/pl/edu/zut/mad/schedule/model/inner/Schedule.java

```diff
@@ -88,4 +88,28 @@
     public String getDay() {
         return day.trim();
     }
+
+    public enum Fields {
+        SURNAME("surname"),
+        ROOM("room"),
+        SUBJECT("subject"),
+        SEMESTER("semester"),
+        FACULTY("faculty"),
+        FACULTY_ABBREVIATION("facultyAbbreviation"),
+        FIELD_OF_STUDY("fieldOfStudy"),
+        FORM("form"),
+        DATE_FROM("dateFrom"),
+        DATE_TO("dateTo"),
+        DATE("date");
```

Comment:
I think key property could be replaced by using `Enum` method `name` or `toString`.

Reply:
That won't work. Values are different from enum field names. Also this way values can be easy modified if names in our model changes.

Reply:
You are right 👌